### ISSUE TITLE
Detected instantly after download

### ISSUE THREAD

**Issue by yegord69** [UnamSanctam/UnamBinder#26]
I tried to download the newest veriosn of the binder from the releases page, but once downloaded WD instantly detects it, is that normal?

**Comment by UnamSanctam** [comment]
Yes of course, I'm not going to try and obfuscate/crypt the builder itself.
